### ISSUE TITLE
Need to capture type of swab

### ISSUE THREAD

**Issue by aliciabeckett-gov** [CDCgov/prime-central#106]
**Describe your feature request in this format: As a __(persona)___ I want to ___(action or feature)__ so that ___(expected outcome)__**
A required field to capture is the type of swab for the test. Some tests like the BinaxNow only use nasal swab, while others like the Sofia and Veritor can use either a nasal swab or nasopharyngeal swab. 

**Describe the solution you'd like and the specific acceptance criteria to be completed**
-Research to understand where it would make sense to capture this data (time of test vs settings)
-Whether our pilot partners have multiple types of swabs
-Design for how this data would be captured in our app
-Once complete, a new ticket should be created for engineering

**Describe alternatives you've considered**
-If we can guarantee there will only be nasal swabs in the future this won't be an issue. 
-I'm working to see how important this is and if we can wait on it or skip it (if nasopharyngeal swab are becoming a thing of the past)

**Additional context**
-Asked Jason Hall about this. His response was "Specimen source should be captured, if possible."

**Comment by willusds** [comment]
Default to nasal swab, need to capture if any site uses nasal pharyngeal
